2019/kw14/GitHub issue pullrequest templates (#360)
Human-Connection/Human-Connection: .github/ISSUE_TEMPLATE/question.md

---

> name: 💬 Question

Reviewer: @ulfgebhardt I would not like to encourage people to submit questions via issues. For that we have Discord: https://discordapp.com/invite/6ub73U3

Author: I disagree - discord is chaotic and for chatting - while a question raised here has a more permanent nature. Therefore I suggest to implement it. A valid strategy to deal with questions would be
a: link to a part of the documentation
b: create documentation and then link

Author: On a side note: The Discord invite is stated in the template to let people think about hooking up there to get their question answered.

Reviewer: If you disagree, I'm fine with it. Let's merge this and observe how many questions we get through the template. If we are unhappy with the outcome (stale issues, unclear definition-of-done) we can remove that issue template any time.

Reviewer 2: I like the points that @ulfgebhardt brought up, I am a little nervous it could get out of hand as @roschaefer maybe is as well, but I think it's a nice compromise to deal with the issue if and when it comes up, not before

> about: If you need help understanding the HumanConnection.

Reviewer: ```suggestion
about: If you need help to understand Human Connection.
```

Reviewer 2: Actually I think grammatically it's better `need help understanding`, but I would also suggest to lose `the` 
Suggested: `If you need help understanding Human Connection`

Author: will adopt!

---
name: 💬 Question

> about: If you need help understanding HumanConnection.

Reviewer: @ulfgebhardt sorry for nitpicking: You forgot to add a space. @Hardy-Peaceglobal already told be about how to correctly spell it:

Wrong:
"HumanConnection"
"Human-Connection"
"Human connection"
"HC"

Correct:
**"Human Connection"**
